lambda_functions/analyzer/analyzer_aws_lib.py

```diff
@@ -210,6 +208,7 @@ def _create_new_entry(self, binary: BinaryInfo, analyzer_version: int) -> None:
             'S3Metadata': binary.s3_metadata,
             'S3Objects': {binary.s3_identifier}
         }
+        LOGGER.info('Creating new entry %s', item)
```

Comment:
Can you confirm the s3_metadata doesn't include sensitive data? We are logging item information which includes more info.

Reply:
Good point - for example, CarbonBlack adds the original filepath to the S3 metadata.

I've changed it to only log the full item if it fails to save to the table